opm/simulators/wells/WellInterface_impl.hpp

```diff
@@ -212,15 +209,7 @@ namespace Opm
             } else {
                 to = Well::ProducerCMode2String(ws.production_cmode);
             }
-            well_control_log_.push_back(from);
-            std::ostringstream ss;
-            ss << "    Switching control mode for well " << this->name()
-               << " from " << from
-               << " to " <<  to;
-            if (cc.size() > 1) {
-               ss << " on rank " << cc.rank();
-            }
-            deferred_logger.info(ss.str());
```

Comment:
Please keep this, but using debug(), not info(), then we still get the appropriate switch messages on each iteration in the debug file (only).